packages/core/src/hydration/api.ts

```diff
@@ -162,6 +162,8 @@ export function withDomHydration(): EnvironmentProviders {
         // be replaced with a tree-shakable alternative (e.g. `isServer`
         // flag).
         if (isBrowser() && inject(IS_HYDRATION_DOM_REUSE_ENABLED)) {
+          verifySsrContentsIntegrity();
+
```

Comment:
nit: we can remove an extra line here
```suggestion
          verifySsrContentsIntegrity();
```